package.json

```diff
@@ -25,12 +25,14 @@
   ],
   "dependencies": {
     "axios": "^1.6.6",
+    "fs": "^0.0.1-security",
```

Comment:
This shouldn't be here (be careful with this). Especially since native modules like FS are already part of node, no install needed